bodhi/server/models.py

```diff
@@ -1602,6 +1602,8 @@ class Update(Base):
             Greenwave integration was not enabled when the update was created.
         compose (Compose): The :class:`Compose` that this update is currently being composed in. The
             update is locked if this is defined.
+        from_tag (unicode): The koji tag from which the list of builds was
```

Comment:
```suggestion
        from_tag (str): The koji tag from which the list of builds was
```
Since Bodhi is Python 3 now the unicode type does not exists anymore :-)

Reply:
Yeah, I noticed that too while working on the CLI part of this. I'll add a tidying up commit to remove 
mentions of `unicode` and `basestr` and let the topical changes in the PR be based on that.

Reply:
```suggestion
        from_tag (str): The koji tag from which the list of builds was
```